### PR TITLE
Updated scheduling api documentation with recurring flags

### PR DESCRIPTION
**Pull request by Antall**

### CODE REVIEW

File: _includes/docs/2.5/ajax-api.html [hunk 1]
```diff
@@ -1075,7 +1121,14 @@ <h3 id="api-schedule-a-flow">Schedule a Flow</h3>
   "status" : "error"
 }
 </pre>
-
+An example failure response for invalid schedule period:
+<pre class="code">
+{
+  "message" : "PROJECT_NAME.FLOW_NAME scheduled.",
+  "error" : "Invalid schedule period unit 'A",
+  "status" : "success"
```
Comment (erwa) [inline _includes/docs/2.5/ajax-api.html]: Even though there's an error, the status is still "success"?

Comment (Antall) [inline _includes/docs/2.5/ajax-api.html]: That is true.
```
vagrant@vagrant-ubuntu-trusty-64:/vagrant/squiddly$ curl -k http://localhost:8081/schedule -d "ajax=scheduleFlow&is_recurring=on&period=15A&projectName=Guzzlr&flow=8_nine_east_wine_emporium&projectId=5&scheduleTime=12,00,am,UCT&scheduleDate=01/01/2015" -b azkaban.browser.session.id=3825dc5e-c82e-4eef-be50-1365097d637c
{
  "message" : "Guzzlr.8_nine_east_wine_emporium scheduled.",
  "error" : "Invalid schedule period unit 'A",
  "status" : "success"
}
vagrant@vagrant-ubuntu-trusty-64:/vagrant/squiddly$
```

Comment (erwa) [inline _includes/docs/2.5/ajax-api.html]: Ah, that's unfortunate. I've opened https://github.com/azkaban/azkaban/issues/407 for this issue.

**Comment by erwa** [comment]
Looks good to me.  Thanks for updating the documentation!

**Pull request merged**